README.md

```diff
@@ -1 +1,50 @@
-# Portfolio
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Portfolio: setup and mobile version skeleton
+
+> THis project is about replicate the figma template 3.
+
+## Built With
+
+- HTML
+- CSS
+- Linters
+  -VSC
+  -GitHUB
+
+## Getting Started
+
+To get a local copy up and running follow these simple example steps:
+Download the project, open with VSC or anorther text editor to edit, and open the file index.html if you only want to watch the web page.
+
+### Prerequisites
+
+Visual code studio, browser.
+
+## Authors
+
+👤 **Richard Burneo**
+
+- GitHub: [@pirc25](https://github.com/pirc25)
+- Twitter: [@RichardBurneo](https://twitter.com/RichardBurneo)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/richard-burneo-aguilera-1277aa203/)
+
+👤 **Author2**
+
+- GitHub: [@githubhandle](https://github.com/githubhandle)
+- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
+
```

Comment:
- [ ] Kindly remove this second author information since it's only one person that worked on it.

Reply:
Thank you for your review. This is done.